st/clitests/awss3api_spec.py

```diff
@@ -1841,6 +1853,335 @@ def get_md5_sum(fname):
 #     .execute_test()\
 #     .command_is_successful()
 
+#*********************List Object Versions***************************
+#******** Create Bucket ********
+AwsTest('Aws can create bucket').create_bucket("versionlistbucket").execute_test().command_is_successful()
+
+AwsTest('Aws can enable versioning on bucket').put_bucket_versioning("versionlistbucket", "Enabled")\
+  .execute_test().command_is_successful()
```

Comment:
Can you format it the same way as the above tests, one line per command? One line per command is more consistent and easier to read.

Reply:
Done